examples/xamarin/BleMicrocontrollerSample/BleMicrocontrollerSample.Android/BleMicrocontrollerSample.Android.csproj

```diff
@@ -2,6 +2,9 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <IntermediateOutputPath>C:\Projects\MyApp</IntermediateOutputPath>
+    <AssemblyName>BleMicroControllerSample.Android</AssemblyName>
+    <RootNamespace>BleMicroControllerSample.Droid</RootNamespace>
+    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
```

Comment:
Why is this changed here and not in the next `PropertyGroup` as in the rest of examples? I would remove this block.